lib/ProductOpener/Food.pm

```diff
@@ -87,6 +87,8 @@ BEGIN {
 
 		&check_nutriscore_categories_exist_in_taxonomy
 
+		&get_nutrient_unit
```

Comment:
I think it would be great if you could add a unit test in [food.t](https://github.com/openfoodfacts/openfoodfacts-server/blob/bf084bbe8ce263cea2eee7f53dcae634fcf1f2eb/tests/unit/food.t)